draft-ietf-cats-usecases-requirements.md

```diff
@@ -913,8 +956,8 @@ informative:
    transfer (e.g., when metrics change significantly).  So in those
    situations:
 
-   R11:
-   : Session as well as service continuity MUST be maintained.
+   R15: session as well as service continuity MUST be maintained when
```

Comment:
I don't parse this requirement

Reply:
same question here. Shall we just say service continuity?

Reply:
Isn't session continuity that is important, i.e., the instance affinity?

Reply:
Agree on that! Given that CATS is about selecting the right instance contact point, I think we should converge to use 'instance affinity', i.e., the affinity of one needed decision at time t to a previous decision at time t-eps.